diff --git a/make-linux.mk b/make-linux.mk
--- a/make-linux.mk
+++ b/make-linux.mk
@@ -335,6 +335,6 @@ endif
 
 # ARM32 hell -- use conservative CFLAGS
-ifeq ($(ZT_ARCHITECTURE),3)
+ifeq (0,3)
 	ifeq ($(shell if [ -e /usr/bin/dpkg ]; then dpkg --print-architecture; fi),armel)
 		override CFLAGS+=-march=armv5t -mfloat-abi=soft -msoft-float -mno-unaligned-access -marm
 		override CXXFLAGS+=-march=armv5t -mfloat-abi=soft -msoft-float -mno-unaligned-access -marm
@@ -359,8 +359,8 @@ ifeq ($(ZT_USE_ARM32_NEON_ASM_CRYPTO),1)
 endif
 
 # Position Independence
-override CFLAGS+=-fPIC -fPIE
-override CXXFLAGS+=-fPIC -fPIE
+#override CFLAGS+=-fPIC -fPIE
+#override CXXFLAGS+=-fPIC -fPIE
 
 # Non-executable stack
 override LDFLAGS+=-Wl,-z,noexecstack
